HorizontalPodAutoscaler/README.md

```diff
@@ -0,0 +1,51 @@
+To create the Deployment, apply the nginx.yaml manifest:
```

Comment:
I tried this out, but the hpa doesn't fully work because this requires the metrics server to be there too. On docker-desktop I followed this: https://github.com/kubernetes-sigs/metrics-server#deployment but ran into other problems where the pod metrics are not available. Is this something you've solved, and can you supply those steps too?

Reply:
Sorry for delay - needed some time for checks and troubleshooting. I created 3 nodes of Kubernetes on DigitalOcean, copied components.yaml from https://github.com/kubernetes-sigs/metrics-server and added next strings to in args section:
"- --kubelet-insecure-tls
- --kubelet-preferred-address-types=InternalIP"
I agree - this is not secure and not good for production deployments. During troubleshooting of this issue I found that now using "kubelet-insecure-tls" in Kubernetes on DigitalOcean is not needed anymore - https://github.com/digitalocean/DOKS/issues/2
But I still cannot make HPA work correctly without "--kubelet-insecure-tls", it works only with both options.
I'll test on a new cluster on DO and on a local server and add needed changes to my examples.